dynamic_rest_client/client.py

```diff
@@ -157,6 +157,11 @@ def __init__(
                 self._use_token(token)
             if cookie:
                 self._use_cookie(cookie)
+            self._authenticate(raise_exception=False)
```

Comment:
`_use_token` and `_use_cookie` no longer modify `_authenticated`. Only `_login` does.